test/ensembl/ensembl.fetchAndLoadById.test.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs');
+const path = require('path');
+const fetchMock = require('node-fetch');
+const { fetchAndLoadById } = require('../../src/ensembl');
+const { ApiConnection } = require('../../src/graphkb');
+
+// Mock node-fetch calls to external APIs
+jest.mock('node-fetch', () => require('fetch-mock-jest').sandbox()); // eslint-disable-line global-require
+const extApiData = JSON.parse(fs.readFileSync(path.join(__dirname, '../data/external_api_responses.json')));
+Object.entries(extApiData).forEach(([url, respBody]) => {
+    fetchMock.get(url, respBody);
+});
+
+// Mock ApiConnexion class with overrided request function
+class MockApiConnection extends ApiConnection {
+    constructor() {
+        super();
+        this.mockFile = path.join(__dirname, '../data/ensembl_fetchAndLoadById_requests.json');
```

Comment:
as discussed in zoom would just be good to add a 1 liner here mentioning why the order of the items in the file is important etc